### ISSUE TITLE
Buttons of CreateTracksActivity2

### ISSUE THREAD

**Issue by somecookie** [somecookie/runPHARAA#171]
We should change the buttons of CreateActivity2 that is currently using the ugly defaults buttons and we could be a bit more thoughtful about the design of this activity.

**Comment by PeterKrcmar0** [comment]
Totaly agree ! We can do a custom design for the buttons and apply it to all. I've done it before so if you need help, tell me.

**Comment by QlimaxXael** [comment]
Good idea ! I'll do it for my "UX improvements" part, that should be not a problem. I'll come to you if I have any problem.
